File: WPFApplication3/Polecenie2.pas
namespace WPFApplication3;

interface

type
  Polecenie2  = public class(System.Windows.Input.ICommand)
  private
  protected
  public
		event CanExecuteChanged: EventHandler;
		method Execute(parameter: Object);
		method CanExecute(parameter: Object): Boolean;
  end;

implementation

method Polecenie2.Execute(parameter: Object);
begin
	await ClassLibrary1.Class1.Test();
	await ClassLibrary1.Class1.Test();
end;

method Polecenie2.CanExecute(parameter: Object): Boolean;
begin
  result := true;
end;

end.

File: WpfApplication2/Polecenie2.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPFApplication2
{
    class Polecenie2 : System.Windows.Input.ICommand
    {
        public event EventHandler CanExecuteChanged;

        public bool CanExecute(object parameter)
        {
            return true;
        }

        public async void Execute(object parameter)
        {
            await ClassLibrary1.Class1.Test();
            await ClassLibrary1.Class1.Test();
        }
    }
}
